files/build_templates/sonic_debian_extension.j2

```diff
@@ -90,6 +90,15 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/ifupdown2_*.deb || \
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/iptables_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install a more recent version of ntp  (and its dependencies via 'apt-get -y install -f')
+sudo dpkg --root=$FILESYSTEM_ROOT --force-confdef -i $debs_path/ntp_*.deb || \
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
+
+# Modifty ntp default configuration: disable initial jump (add -x), and disable
+# jump when time difference is greater than 1000 seconds (remove -g).
+sudo sed -i "s/NTPD_OPTS='-g'/NTPD_OPTS='-x'/" $FILESYSTEM_ROOT/etc/default/ntp
```

Comment:
check files/image_config/ntp/ntp-config.sh, we have it. why duplicate here?

Reply:
Thats true. I will back-off this change.